test/manual/algo/README.md

```diff
@@ -89,7 +89,7 @@ go run ./test/computations/main.go ./test/manual/algo/addition.py public.pem fal
 ```
 
 ```bash
-./build/cocos-cli algo ./test/manual/algo/addition.py ./private.pem -a python
+./build/cocos-cli algo ./test/manual/algo/addition.py ./private.pem -a python --args="100,20"
```

Comment:
Can we modify the algo to take something like

```bash
python addition.py --a 100 --b 20
```

As it seems the best use case for using arguments